CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Bump: sentry-cocoa to v6.2.1 (#360)
 * Fix: missing event.origin and event.environment tags for non-native events on iOS (#369)
 * Fix: Read all environment variables in sentry_dart (#375)
+* Fix: Set `SentryOptions.debug` in sentry_dart (#376)
```

Comment:
```suggestion
* Fix: Set `SentryOptions.debug` in `sentry` (#376)
```

`sentry_dart` does not exist, let's call it `sentry` only, fix the PRs title too.